ajax/libs/zepto/package.json

```diff
@@ -20,6 +20,12 @@
       "files": [
         "**/*"
       ]
+    },
+    {
+      "basePath": "src",
+      "files": [
+        "!(zepto)*"
```

Comment:
Then zepto.js can be reserved. Suppose this zepto.js is different from the one under `dist`.